gengokit/template/NAME-service/handlers/server/server_handler.gotemplate

```diff
@@ -18,33 +15,26 @@ import (
 
 
 // NewService returns a naïve, stateless implementation of Service.
-func NewService() Service {
+func NewService() pb.{{GoName .Service.GetName}}Server {
 	return {{.PackageName}}Service{}
 }
 
 type {{.PackageName}}Service struct{}
 
-{{with $Executor := .}}
-{{with $PackageName := $Executor.PackageName}}
-
-{{range $i := $Executor.Service.Methods}}
-// {{$i.GetName}} implements Service.
-func (s {{$PackageName}}Service) {{$i.GetName}}(ctx context.Context, in *pb.{{GoName $i.RequestType.GetName}}) (*pb.{{GoName $i.ResponseType.GetName}}, error){
-	_ = ctx
-	_ = in
-	response := pb.{{GoName $i.ResponseType.GetName}}{
-		{{range $j := $i.ResponseType.Fields -}}
-		// {{GoName $j.GetName }}: 
-		{{end -}}
-	}
-	return &response, nil
-}
-{{end}}
-{{- end}}
+{{with $tE := .}}
+	{{with $PackageName := $tE.PackageName}}
```

Comment:
the `tE` seems slightly odd on the eyes, I keep thinking it's a typo. Not necessarily important that it change, but if I came across this I'd be like wat, and if I where given the chance to refactor this I'd definitely do it.